dev/remsh-tls

```diff
@@ -26,4 +26,4 @@ fi
 NAME="remsh$$@$HOST"
 NODE="node$NODE@$HOST"
 rootdir="$(cd "${0%/*}" 2>/dev/null; echo "$PWD")"
-erl -name $NAME -remsh $NODE -hidden -proto_dist inet_tls -ssl_dist_optfile "${rootdir}/couch_ssl_dist.conf"
+erl -name $NAME -remsh $NODE -hidden -proto_dist inet_tls -ssl_dist_optfile "${rootdir}/../src/couch_dist/certs/out/couch_dist.conf"
```

Comment:
consider passing the location in (env var, perhaps) rather than having to do `../src`.